### PR TITLE
New package: shc-4.0.3_1

### PR DESCRIPTION
**Pull request by lilmike**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/shc/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+license="GPL-3.0-or-later"
+homepage="https://github.com/neurobin/shc"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum="7d7fa6a9f5f53d607ab851d739ae3d3b99ca86e2cb1425a6cab9299f673aee16"
+
+pre_check() {
+	# Remove ash from the list of shells because it's not a package on void.
+	sed -i "s;'/bin/ash';;g" "$wrksrc/test/ttest.sh"
+	# Also remove dash, since installing it as a host dependency conflicts with chroot-bash
+	sed -i "s;'/bin/dash';;g" "$wrksrc/test/ttest.sh"
```
Comment (ericonr) [inline srcpkgs/shc/template:18]: Please use `vsed`.

File: srcpkgs/shc/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'shc'
+pkgname=shc
+version=4.0.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="tcsh rc ksh zsh"
```
Comment (ericonr) [inline srcpkgs/shc/template:6]: Aren't these probably `checkdepends` instead of `hostmakedepends`?

**Review by ericonr — COMMENTED** [review]
Would be nice if upstream could release new versions, they seem reasonably active.

**Comment by lilmike** [comment]
I've fixed all those issues. Thanks!





On Tue, Mar 09, 2021 at 08:29:30PM -0800, Érico Nogueira Rolim wrote:
>@ericonr commented on this pull request.
>
>Would be nice if upstream could release new versions, they seem reasonably active.
>
>> +	sed -i "s;'/bin/ash';;g" "$wrksrc/test/ttest.sh"
>+	# Also remove dash, since installing it as a host dependency conflicts with chroot-bash
>+	sed -i "s;'/bin/dash';;g" "$wrksrc/test/ttest.sh"
>
>Please use `vsed`.
>
>> @@ -0,0 +1,19 @@
>+# Template file for 'shc'
>+pkgname=shc
>+version=4.0.3
>+revision=1
>+build_style=gnu-configure
>+hostmakedepends="tcsh rc ksh zsh"
>
>Aren't these probably `checkdepends` instead of `hostmakedepends`?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/void-linux/void-packages/pull/29041#pullrequestreview-608320845

-- 
Michael Taboada, Creator of Games. Does that make me a god, well, sorta.
My pgp key: 79BC390E8864CD71305D4DD606CD2197A6EE5ED7
My website: https://michaels.world
2MB website: https://2mb.games
Don't believe everything you read on the internet. I might not be real, you might not be real, and this email certainly isn't real.
Sent from my hall closet...
dedicated server

**Pull request closed without merging**